extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/runtime/exporter/OtlpExporterConfig.java

```diff
@@ -5,11 +5,9 @@
 import java.util.Optional;
 import java.util.OptionalInt;
 
-import io.quarkus.runtime.annotations.ConfigGroup;
 import io.smallrye.config.WithDefault;
 import io.smallrye.config.WithName;
 
-@ConfigGroup
```

Comment:
I know it's not mandatory but I would say it's a good practice to keep the annotation, given the annotation processor is annotation driven.
They get discovered anyway when analyzing the config but this makes it discovered right away by the AP.

Reply:
When this annotation is present in a super class relative to the mapping class, we do generate duplicate entries of the configuration, because we handle `@ConfigGroup` as a nested element and not an an inherited element.

We could also fix that in the code, but I think that super classes are not config groups, so it is a bit misleading to add that annotation.